django_tables2_column_shifter/tables.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, *args, **kwargs):
             self.template_name = self.shifter_template
         else:
             self.template = self.shifter_template
+        """By default all of the columns are hideable"""
```

Comment:
For one-line comment use `#`
multi line strin `"""` should be usage for docstring (for dscription class or method)

Reply:
Wanted to be consistent with comment above it:

"""Override init for set shifter template"""